app/views/users/show.html.erb

```diff
@@ -1,4 +1,7 @@
-<h2><%= @user.username %></h2>
+<div class='user-crawler'>
+	<img src='<%= get_absolute_image_url(@user.small_avatar_url) %>' alt='<%= t('user.image_alt', username: @user.username) %>' title='<%= @user.username %>' />
```

Comment:
we don't want an absolute image url here, we want a CDN url

Reply:
do we have any helper function to get a CDN URL? @SamSaffron

Reply:
used `UrlHelper.local_cdn_url`, let me know if there is an issue with it.